diesel/src/sqlite/connection/row.rs

```diff
@@ -16,11 +17,19 @@ pub struct SqliteRow<'stmt, 'query> {
 pub(super) enum PrivateSqliteRow<'stmt, 'query> {
     Direct(StatementUse<'stmt, 'query>),
     Duplicated {
+        // starting point to create the OwnedSqliteRow
```

Comment:
```suggestion
```

We don't need that work in process comment anymore, right?

Reply:
Cleaned